internal/pkg/service/stream/storage/statistics/provider_test.go

```diff
@@ -1,22 +1,28 @@
 package statistics_test
 
 import (
+	"bytes"
 	"context"
 	"testing"
 	"time"
 
 	"github.com/benbjohnson/clock"
+	"github.com/keboola/go-client/pkg/keboola"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"go.etcd.io/etcd/client/v3/concurrency"
 
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
+	commonDeps "github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/common/utctime"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/level"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition/key"
+	dependencies "github.com/keboola/keboola-as-code/internal/pkg/service/stream/dependencies"
```

Comment:
the left part could be omitted.

Reply:
Ok

Reply:
[fix: Review fixes](https://github.com/keboola/keboola-as-code/pull/1820/commits/03b95f2f3ba09bd1292215856802fa207b2b71aa)